src/charts/c3/c3.mixin.js

```diff
@@ -0,0 +1,57 @@
+angular.module('patternfly.charts').factory('ChartsMixin', function() {
+    'use strict';
+
+    // Donut Chart Defaults
+    var getDefaultDonut = function(title) {
+        return {
+            title: title,
+            label: {
+                show: false
+              },
+              width: 10
+            };
+      };
+
+    var getDefaultDonutSize = function() {
```

Comment:
Sorry, I'm not understanding this request.  Could you please provide an example?
Thanks